### PR TITLE
Added tests for preferences

### PR DESCRIPTION
**Pull request by hpohekar**
image

### CODE REVIEW

File: tests/test_preferences.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import ansys.fluent.core as pyfluent
+
+
+def test_solver_verbosity():
```
Comment (seanpearsonuk) [inline tests/test_preferences.py:4]: @hpohekar have a look at other tests.  You will need to use a _fixture_ for the launch part. Have a look at test_creatable.py for instance.

Comment (hpohekar) [reply tests/test_preferences.py]: Done. Thank you.

Comment (hpohekar) [reply tests/test_preferences.py]: image

**Review by seanpearsonuk — APPROVED** [review]

**Review by prmukherj — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

File: tests/test_preferences.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    solver = pyfluent.launch_fluent(mode="solver")
```
[the remark this answers is not in the document]
Comment (hpohekar) [reply tests/test_preferences.py]: Yes, we have used fixture here.

image

**Pull request merged**